stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.3.1
+version: 0.5.0
```

Comment:
Only because I saw the 4.x version from @jwhitcraft's PR and didn't want to conflict.  I thought the new functionality added was significant enough to warrant a 0.x.0 increase versus 0.0.x increase.  Happy to change to what you think is best.